roles/edpm_iscsid/templates/iscsid.yaml.j2

```diff
@@ -1,13 +1,37 @@
-image: {{ edpm_iscsid_image }}
-net: host
-privileged: true
-restart: always
-{% if edpm_iscsid_healthcheck %}
-healthcheck:
-  test: /openstack/healthcheck
-  mount: /var/lib/openstack/healthchecks/iscsid
-{% endif %}
-volumes:
-  {{ edpm_container_standalone_common_volumes | default([]) + edpm_iscsid_volumes }}
-environment:
-  KOLLA_CONFIG_STRATEGY: COPY_ALWAYS
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    bind-mount-options: /etc/iscsi:z
+  creationTimestamp: "2024-12-09T02:00:14Z"
+  labels:
+    app: iscsid
+  name: iscsid
```

Comment:
would it make sense to move some of the above boilerplate to the edpm_container_quadlet role? Could be populated with the service name, etc.

Reply:
Yeah, that's a good idea. Probably a little more templating required for things like `hostNetwork` I guess. I'll have to check the other templates to see what other things are different. But should be easy enough to template the rest of it.